DropDownMenu.swift

```diff
@@ -79,7 +79,7 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		}
 	}
 	open var backgroundAlpha = CGFloat(1)
-	
+    var newTableViewFrame : CGRect
```

Comment:
Would be be nice to get the same indentation than the previous lines (with tabs)